bin/create-exercise

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+die() { echo "$*" >&2; exit 1; }
+
+if [[ $PWD != $(realpath "$(dirname "$0")/..") ]]; then
+    die "You must be in the track root directory."
+fi
+if [[ -z $1 ]]; then
+    die "usage: $0 exercise_slug"
+fi
+
+slug=$1
+
+existing=$( jq --arg slug "${slug}" '.exercises.practice[] | select(.slug == $slug)' config.json )
+if [[ -n ${existing} ]]; then
+    die "${slug} already exists in config.json"
+fi
+
+if [[ -z $author ]]; then
+    echo
+    read -rp "What's your github username? " author
+fi
+
+read -p "What's the difficulty for ${slug}? " difficulty
+
+bin/fetch-configlet
+bin/configlet create --practice-exercise "${slug}" --author "${author}" --difficulty "${difficulty}"
+
+cp exercises/practice/hello-world/.docs/hints.md exercises/practice/${slug}/.docs/hints.md
+
+touch exercises/practice/${slug}/runner.mips
+touch exercises/practice/${slug}/impl.mips
+touch exercises/practice/${slug}/.meta/example.mips
+touch exercises/practice/${slug}/.docs/instructions.append.md
```

Comment:
Is this really necessary? I would think configlet create creates these files.

Reply:
Fixed. We only need the `.docs/instructions.append.md`  (register assignment)